frontend/src/components/header/Header.jsx

```diff
@@ -1,44 +1,90 @@
-import React from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import { ReactComponent as Logo } from 'assets/images/logo.svg';
+import { ReactComponent as Menu } from 'assets/images/menu.svg';
+import { ReactComponent as Close } from 'assets/images/close.svg';
```

Comment:
also please verify that this one exists

Reply:
alright

Reply:
its breaking insides assets/images so i have moved it to assets/icons